src/components/SubjectInterest.vue

```diff
@@ -99,6 +97,9 @@ export default {
         }
       }, 100)
     }
+  },
+  created: function () {
```

Comment:
Can you give this a try?

hook into `mounted` instead of `created`, and try removing the setInterval.  https://alligator.io/vuejs/component-lifecycle/

Alternately, you could try creating  a global custom directive, since this focus behavior is used so often https://vuejs.org/v2/guide/custom-directive.html